fedimint-bitcoind/src/bitcoincore_rpc.rs

```diff
@@ -215,8 +215,7 @@ where
             self.0
                 .estimate_smart_fee(confirmation_target, Some(EstimateMode::Conservative))
                 .map_err(anyhow::Error::from)
-        })
-        .expect("Bitcoind returned an error") // TODO: implement retry logic in case bitcoind is temporarily unreachable
```

Comment:
Is this retry logic still needed?

Reply:
Nope we wrap all bitcoin rpc calls in retries already.